java-kotlin-aot/README.md

```diff
@@ -12,44 +12,27 @@ This repository contains the code for a demo application for [GraalVM](graalvm.o
 This is a simple Java/Kotlin application, where a Java method accesses a String from Kotlin and calls a Kotlin function, which later accesses a String from a Java class.
 This example demonstrates how easy it is to interop between Java and Kotlin.
 
-1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$JAVA_HOME` and add `$JAVA_HOME/bin` to the `PATH` environment variable.
-
-  On Linux:
-  ```bash
-  export JAVA_HOME=/home/${current_user}/path/to/graalvm
-  export PATH=$JAVA_HOME/bin:$PATH
-  ```
-  On macOS:
-  ```bash
-  export JAVA_HOME=/Users/${current_user}/path/to/graalvm/Contents/Home
-  export PATH=$JAVA_HOME/bin:$PATH
-  ```
-  On Windows:
+1. Download and install the latest GraalVM JDK with Native Image using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader). 
   ```bash
-  setx /M JAVA_HOME "C:\Progra~1\Java\<graalvm>"
-  setx /M PATH "C:\Progra~1\Java\<graalvm>\bin;%PATH%"
+  bash <(curl -sL https://get.graalvm.org/jdk) 
   ```
 
-2. Install [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/#install-native-image) by running:
+2. Download or clone the repository and navigate into the `java-kotlin-aot` directory:
   ```bash
-  gu install native-image
+  git clone https://github.com/graalvm/graalvm-demos
   ```
-
-3. Download or clone the repository and navigate into the `java-kotlin-aot` directory:
   ```bash
-  git clone https://github.com/graalvm/graalvm-demos
   cd graalvm-demos/java-kotlin-aot
   ```
 
 ## Build the Application
 
-Before running this example, you need to build the application:
+For build the application use this command:
```

Comment:
For build the application use this command > To build the application, use this command